Assignment4/scipi0n-180463/task06.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1sOG5w44E98LgM4Ku1GNJ5kAEpsbODUJR
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't run in Python. You should comment out this line. Same for the rest of the tasks